src/main/java/org/gridsuite/modification/server/modifications/VoltageInitModification.java

```diff
@@ -26,14 +26,16 @@
 public class VoltageInitModification extends AbstractModification {
     private final VoltageInitModificationInfos voltageInitModificationInfos;
 
-    private static final String GENERATOR_KEY = "GeneratorModifications";
-    private static final String GENERATOR_NAME = "Generator ${generatorId}";
-    private static final String STATIC_VAR_COMPENSATOR_KEY = "StaticVarCompensatorModifications";
-    private static final String STATIC_VAR_COMPENSATOR_NAME = "Static var compensator ${compensatorId}";
-    private static final String VSC_CONVERTER_STATION_KEY = "VscConverterStationModifications";
-    private static final String VSC_CONVERTER_STATION_NAME = "Vsc converter station ${converterId}";
-    private static final String SHUNT_COMPENSATOR_KEY = "ShuntCompensatorModifications";
-    private static final String SHUNT_COMPENSATOR_NAME = "Shunt compensator ${compensatorId}";
+    private static final String GENERATORS_KEY = "GeneratorsModifications";
+    private static final String GENERATORS_NAME = "Generators";
+    private static final String TRANSFORMERS_KEY = "TransformersModifications";
+    private static final String TRANSFORMERS_NAME = "Transformers";
+    private static final String STATIC_VAR_COMPENSATORS_KEY = "StaticVarCompensatorsModifications";
+    private static final String STATIC_VAR_COMPENSATORS_NAME = "Static var compensators";
+    private static final String VSC_CONVERTER_STATIONS_KEY = "VscConverterStationsModifications";
+    private static final String VSC_CONVERTER_STATIONS_NAME = "Vsc converter stations";
+    private static final String SHUNT_COMPENSATORS_KEY = "ShuntCompensatorsModifications";
+    private static final String SHUNT_COMPENSATORS_NAME = "Shunt compensators";
```

Comment:
These constants are use only a single time, can be inline in code where it's used.
```suggestion
```